nes-nebuli/src/NebuLIStarter.cpp

```diff
@@ -191,7 +191,9 @@ int main(int argc, char** argv)
             auto singleNodeWorkerConfig = NES::loadConfiguration<NES::SingleNodeWorkerConfiguration>(singleNodeArgC, singleNodeArgV.data())
                                               .value_or(NES::SingleNodeWorkerConfiguration{});
 
-            queryManager = std::make_shared<NES::EmbeddedWorkerQueryManager>(singleNodeWorkerConfig);
+            /// The embedded worker does not actually expose a grpc server so the grpc address is ignored. It has to be a valid URL.
+            queryManager = std::make_shared<NES::QueryManager>(std::make_unique<NES::EmbeddedWorkerQuerySubmissionBackend>(
+                NES::WorkerConfig{NES::GrpcAddr("i-dont-exist:0")}, singleNodeWorkerConfig));
```

Comment:
You could remove the struct, as it only wraps the addr.

Reply:
yes, but the struct is growing in the next pr anyways. This is just the backported version, which does not contain the host address yet.